Address Kirsty's comments. (#227)
cfrg/draft-irtf-cfrg-hpke: draft-irtf-cfrg-hpke.md

--- abstract

This document describes a scheme for hybrid public-key encryption (HPKE).

> This scheme provides a variant of public key encryption of arbitrary-sized

Reviewer: ```suggestion
This scheme provides a variant of public-key encryption of arbitrary-sized
```

plaintexts for a recipient public key. It also includes three authenticated
variants, including one which authenticates possession of a pre-shared key,

> and two optional ones which which authenticates possession of a private key.

Reviewer: ```suggestion
and two optional ones which authenticate possession of a KEM private key.
```

## Security Requirements on a KDF {#kdf-choice}

The choice of the KDF for HPKE SHOULD be made based on the security
level provided by the KEM and, if applicable, by the PSK. The KDF

> SHOULD have at least have the security level of the KEM and SHOULD

Reviewer: ```suggestion
SHOULD at least have the security level of the KEM and SHOULD
```

at least have the security level provided by the PSK.

## Security Requirements on an AEAD {#aead-security}

> All AEADs must be IND-CCA2-secure, as is currently true for all AEADs

Reviewer: ```suggestion
All AEADs MUST be IND-CCA2-secure, as is currently true for all AEADs
```
